src/main/io/ledstrip.h

```diff
@@ -153,6 +153,7 @@ typedef struct ledStripConfig_s {
     uint8_t ledstrip_aux_channel;
     ioTag_t ioTag;
     ledStripFormatRGB_e ledstrip_grb_rgb;
+    uint8_t odEnable;
```

Comment:
Normally, the name for new parameters is chosen to match the name of the CLI parameter (i.e. snake_case). This breaks with naming conventions, but simplifies responding to support questions.

Reply:
Changed to `isOpenDrain`, to be consistent with `isOpenDrain` in `beeperConfig_s`. Thoughts?

Reply:
Sounds good to me.